css/styles.css

```diff
@@ -125,6 +219,29 @@ section > ul {
   text-align: center;
 }
 
+@media screen and (min-width: 992px) {
+  .socialContent {
+    justify-content: space-between;
+    display: flex;
+    flex-direction: row;
+    width: 900px;
+  }
+
+  h4 {
+    text-transform: none;
+  }
+
+  .imgNarrow,
+  .imgPlace {
+    display: none;
+  }
+
+  .imgWide,
+  .imgPlaceWide {
+    display: inline;
+  }
+}
```

Comment:
- [ ] Good job adding the desktop version, I noticed that you always specified the breakpoint every time for the desktop version, you can group all the styling together that have the same screen size this will make your code easier to read, manage and less confusing, unless you are using another screen size then you create another breakpoint   👍🏿 

`@media screen and (min-width: 992px) {
/*
Kindly add all desktop styling with same screen size to one block
*/
}`